packages/jimp/README.md

```diff
@@ -80,43 +80,65 @@ If you're using webpack you can set `process.browser` to true and your build of
 
 ## Basic usage
 
-The static `Jimp.read` method takes the path to a file, URL, dimensions, a Jimp instance or a buffer and returns a Promise:
+The static `Jimp.read` method takes the path to a file, URL, a Jimp instance or a buffer and returns a Promise:
 
 ```js
 Jimp.read('./path/to/image.jpg')
   .then(image => {
-    // do stuff with the image
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
   });
-
-Jimp.read(lenna.buffer)
+  
+Jimp.read('http://www.example.com/path/to/lenna.jpg')
   .then(image => {
-    // do stuff with the image
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
+  });
+  
+Jimp.read(jimpInstance)
+  .then(image => {
+    // Do stuff with the image.
+  })
+  .catch(err => {
+    // Handle an exception.
   });
 
-Jimp.read('http://www.example.com/path/to/lenna.jpg')
+Jimp.read(buffer)
   .then(image => {
-    // do stuff with the image
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
   });
+```
+
+In some cases, you need to pass additional parameters alongside with an image's URL. You can pass an object to the `Jimp.read` method:
```

Comment:
`alongside with` choose either `alongside` or `width` and it reads better. they both mean the same thing

Reply:
Thanks. Fixed.